app/controllers/ukrentaroom/UkRentARoomExpensesIntroController.scala

```diff
@@ -40,7 +40,7 @@ class UkRentARoomExpensesIntroController @Inject()(
     implicit request =>
```

Comment:
Looks like taxYear is not used in the onPageLoad() function. If that's the case maybe you can completely remove it.